django/utils/dateformat.py

```diff
@@ -314,7 +314,7 @@ def y(self):
 
     def Y(self):
         "Year, 4 digits; e.g. '1999'"
-        return self.data.year
+        return '%04d' % self.data.year
```

Comment:
Can we add a note to the docstring? e.g.
```python
"""Year, 4 digits with leading zeros; e.g. '1999'."""
```
and to the docs:
https://github.com/django/django/blob/205c36b58fed5a1a0ff462593fc61b58189027d8/docs/ref/templates/builtins.txt#L1370
e.g.
```

``Y``             Year, 4 digits with leading zeros.        ``'0000'``, ...,``'1999'``, ...,``'9999'`` 
```